src/palace/manager/api/axis.py

```diff
@@ -1578,6 +1581,14 @@ def process_one(
                 or ""
             )
 
+            if not self.internal_format and (
+                checkout_format == self.api.AXISNOW or checkout_format == "Blio"
```

Comment:
Minor: Should we add "BLIO" as a constant to [the `Axis360API` class](https://github.com/ThePalaceProject/circulation/blob/05748ac3b5c1abf8672239526e4de7dbd9b5c76e/src/palace/manager/api/axis.py#L201-L200) and reference that here?